projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -187,6 +194,17 @@ describe('AuthService', () => {
       });
     });
 
+    it('should not handle the callback when skipRedirectCallback is true', (done) => {
+      authConfig.skipRedirectCallback = true;
+
+      const localService = createService();
+
+      loaded(localService).subscribe(() => {
+        expect(auth0Client.handleRedirectCallback).not.toHaveBeenCalledTimes(1);
```

Comment:
We might as well just use `toHaveBeenCalled` here. Would this assertion pass if we called `handleRedirectCallback` more than once? Even if not, the usage here is slightly confusing at first glance.

Reply:
Good catch, copy paste issue from the above unit test 😬